lib/core/types.hpp

```diff
@@ -1,7 +1,17 @@
 #pragma once
+#include <stdint.h>
 
 namespace hyped::core {
 
 enum class LowOrHigh { kLow = 0, kHigh };
 
+struct CanFrame
```

Comment:
Is this struct from a datasheet/spec?

Reply:
This is the [SocketCAN spec](https://docs.kernel.org/networking/can.html)